MIG.HomeAutomation/ZWave.cs

```diff
@@ -253,7 +266,7 @@ public bool IsConnected
 
         public object InterfaceControl(MigInterfaceCommand request)
         {
-            ResponseText returnValue = new ResponseText("OK");
+            object returnValue = "OK";
             bool raiseEvent = false;
```

Comment:
Ok, I'll do this. But what was the point of wrapping serialized object representation in ResponseText object?